endoreg_db/services/video_import.py

```
@@ -180,50 +349,186 @@ def _process_frames_and_metadata(self):
+        try:
+            self.logger.info("Attempting fallback video anonymization...")
+            if self.current_video:
+                # Try VideoFile.pipe_2() method if available
+                if hasattr(self.current_video, 'pipe_2'):
+                    self.logger.info("Trying VideoFile.pipe_2() method...")
+                    if self.current_video.pipe_2():
+                        self.logger.info("VideoFile.pipe_2() succeeded")
+                        self.processing_context['anonymization_completed'] = True
+                        return
+                    else:
+                        self.logger.warning("VideoFile.pipe_2() returned False")
```

Pylint (4.0.0)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)